### PR TITLE
fix bug that prevents logging in with newly created users

### PR DESCRIPTION
**Pull request by yunhangc**
added Name for user table and username for auth table to be able to identify the login user

### CODE REVIEW

File: server/api/auth.py [hunk 1]
```diff
@@ -61,17 +61,19 @@ def decorator(func):
...
+    auth_info = await auth_query.fetch_info_by_user_name(
         request.app.config.DB_CONN, request.json.get('id')
     )
     if auth_info is None or auth_info.get('hashed_password') != hashed_pwd:
         raise ApiUnauthorized("Unauthorized: Incorrect user id or password")
     token = utils.generate_apikey(
         request.app.config.SECRET_KEY,
-        request.json.get('id')
+        auth_info.get('user_id')
+        # request.json.get('id')
```
Comment (dcmiddle) [inline server/api/auth.py]: Please remove the commented code. We typically only commit active code and rely on version control for preserving old code.

Comment (yunhangc) [reply server/api/auth.py]: I have missed the comment. Completely agree that no code comment should be checked in. Thanks for pointing this out.

**Comment by yunhangc** [comment]
@dcmiddle X-XSRF-TOKEN is removed accidentally. putting it back.

File: server/db/auth_query.py [hunk 2]
```diff
@@ -17,16 +17,33 @@
...
     auth_info = await r.table('auth').get(user_id).run(conn)
     if auth_info is None:
         raise ApiNotFound(
             "Not Found: "
             "No user with id '{}' exists.".format(user_id)
         )
     return auth_info
+
+async def fetch_info_by_user_name(conn, user_name):
+    auth_info = await r.table('auth').filter(r.row["user_name"] == user_name).coerce_to('array').run(conn) #filter( {user_name: 'manager_a'}).run(conn)
```
[the remark this answers is not in the document]
Comment (yunhangc) [reply server/db/auth_query.py]: this is already removed at the other PR. the other PR is based on this branch.

**Pull request merged**